examples/web_apps/webxr_mobile_demo/style.css

```diff
@@ -0,0 +1,7 @@
+#fps {
```

Comment:
This files seems overkill just for this. You could just specify the style inline in the html and omit this file.

Reply:
Oops, I don't even calculate FPS in the mobile demo. I'll remove this file. Although I do the same thing for the hand demo, so I'll change that.